blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -80,7 +80,12 @@ export class RunRecord extends Record({
     }
 
     getComputedResult() {
-        return this.isCompleted() ? this.result : this.state;
+        if (this.isCompleted()) {
+            return this.result;
+        } else if (this.state === 'PAUSED') { // TODO: remove this after JDL is defined for paused state
```

Comment:
This should be done in ActivityService#bunkerMap. I can do it if you would like.